src/cards/ha-weather-card.js

```diff
@@ -237,16 +241,20 @@ class HaWeatherCard extends
     return typeof item !== 'undefined' && item !== null;
   }
 
-  computeDateTime(data) {
+  computeDate(data) {
     const date = new Date(data);
-    const provider = this.stateObj.attributes.attribution;
-    if (provider === 'Powered by Dark Sky' || provider === 'Data provided by OpenWeatherMap') {
-      return date.toLocaleTimeString(
-        this.hass.selectedLanguage || this.hass.language,
-        { hour: 'numeric' }
-      );
-    }
-    return date.toLocaleDateString(this.hass.selectedLanguage || this.hass.language, { weekday: 'short' });
+    return date.toLocaleDateString(
+      this.hass.selectedLanguage || this.hass.language,
+      { weekday: 'short' }
+    );
+  }
+
+  computeTime(data) {
+    const date = new Date(data);
+    return date.toLocaleTimeString(
+      this.hass.selectedLanguage || this.hass.language,
+      { hour: '2-digit' }
```

Comment:
for which language code and browser does this work btw?

Reply:
'2-digit' works like 'numeric' in my case. Do I need to change it to 'numeric'?

Reply:
well you changed it that's why I ask... seems to be not working all in all

Reply:
Could you specify what exactly is wrong here, please?

Reply:
stackoverflow says you have to use `[]` as language parameter to use '2-digit'

Reply:
Thanks, i tried this. Square brackets removes minutes and seconds but hours still 'numeric'. I will revert '2-digit' to 'numeric'